components/nimbus/src/enrollment.rs

```diff
@@ -726,7 +727,7 @@ impl<'a> EnrollmentsEvolver<'a> {
                 prev_experiments.get(slug).copied(),
                 next_experiments.get(slug).copied(),
                 Some(prev_enrollment),
-                event_store,
+                event_store.clone(),
```

Comment:
Here's another clone of the event store before it is used, since this is 3 functions deep now we have made 3 copies and haven't accessed any of them.